### PR TITLE
Feature/user resizable facets panel

### PR DESCRIPTION
**Pull request by nyanofthemoon**

### CODE REVIEW

**Comment by ciaranschutte** [comment]
Not working locally for me ... no 'crosshair' to resize when I hover over border of sidebar

**Comment by nyanofthemoon** [comment]
@ciaranschutte went with the closeable option like the right panel

File: src/components/FileRepo/AggregationSidebar/AggregationSidebar.js [hunk 1]
```diff
@@ -59,19 +80,20 @@ const AggregationSidebar = compose(injectState, withTheme, withApi)(
     translateSQONValue,
     trackFileRepoInteraction,
     aggregationsWrapperRef = React.createRef(),
+    expanded,
+    setExpanded,
+    test = 1000,
```
Comment (ciaranschutte) [inline src/components/FileRepo/AggregationSidebar/AggregationSidebar.js]: What is this?

File: src/components/FileRepo/AggregationSidebar/AggregationSidebar.js [hunk 2]
```diff
@@ -59,19 +80,20 @@ const AggregationSidebar = compose(injectState, withTheme, withApi)(
...
-                <H2>
-                  <Trans>Filters</Trans>
-                </H2>
-              </AggregationTitle>
-              <TealActionButton
+        <CollapsibleAggregationWrapper
+          {...{ scrollbarWidth, expanded, innerRef: aggregationsWrapperRef }}
+        >
+          <AggregationHeader>
+            {expanded ? <H2>Filter</H2> : null}
```
Comment (ciaranschutte) [inline src/components/FileRepo/AggregationSidebar/AggregationSidebar.js]: Can you join these two conditionals to use only one `expanded`?
eg.
``` javascript
{expanded ? <h2>filter</h2><browseAllButton ......> : null}
```

**Pull request merged**